src/shared/bg_public.h

```diff
@@ -82,6 +81,8 @@ struct glm4rw_impl
 
 #include "engine/qcommon/q_shared.h"
 
+#define BEST_LEN 3.0f * 8192.0f * 8192.0f
```

Comment:
The naming of this makes no sense. What is it supposed to mean?

You could make a constant named like HUGE_DISTANCE, make it 1e10 or 1e15. That's big enough that no one could confuse it with a normal distance, but small enough that it won't cause overflows. It would be good to replace the existing instances of HUGE_QFLT (which are all distances) with that, since HUGE_QFLT is too big and has the overflow risk.

Reply:
I only named it `BEST_LEN` because it was named `bestlen` before, so “not my fault!”. 🤪️

I welcome any better name.

Reply:
Hmm, `HUGE_QFLT` is defined in engine but only used in sgame, so we may move it to the game.

I wonder what can mean `QFLT`.

Also I don't know what this comment means about `MAX_QFLT` (that doesn't exist):

```c++
#define HUGE_QFLT 3e38f // TODO: Replace HUGE_QFLT with MAX_QFLT
```

Reply:
Maybe QFLT means some QFloat like some QBoolean.

Reply:
Maybe the comment about `MAX_QFLT` was not about using the max float value for `MAX_QFLT`, but about giving us a new `max` to use that is not the actual max of the float (so, one that doesn't overflow).